lib/ansiblelint/utils.py

```diff
@@ -763,15 +763,48 @@ def is_playbook(filename):
     return False
 
 
+def get_yaml_files(options=None):
```

Comment:
I think this shouldn't be optional as it's only used in one way in only one place making this assumption premature.

Reply:
Could you please elaborate?
Do you suggest to change this line to `def get_yaml_files(options):`?

Reply:
Exactly. Remove `=None` and the following `is None` if-clause block.

Reply:
done